amex-page.html

```diff
@@ -0,0 +1,271 @@
+                <div class="column">
+                    <a href="url">
+                        <img src="cash-back-card.gif" alt="light blue credit card" style="width:237.5px;height:25px;">                
+                    </a>                
+                </div>
+                <div class="column">
+                    <a href="url">
+                        <img src="co-branded-card.gif" alt="black credit card" style="width:237.5px;height:25px;">                
+                    </a>                
+                </div>                        
+                <div class="column">
+                    <a href="url">
+                        <img src="business-card.gif" alt="blue credit card" style="width:237.5px;height:25px;">                
+                    </a>                
+                </div>
+            </div>                
+            <div class="row">
+                <div class="column">
+                    <img src="info-icon.jpg" alt="an icon of a  circle with a lower letter i in it">
+                </div>            
+                <div class="column">
+                    <img src="chat-box.jpg" alt="an icon of a chat box">
+                </div>
+                <div class="column">
+                    <img src="hexigon.jpg" alt="a hexigon icon with a star in the middle">
+                </div>
+            </div>
+            <div class="row">
+                <div class="column">
+                    <a href="#Covid-19">COVID-19 Support</a>
+                </div>
+                <div class="column">
+                    <a href="#Visit">Visit Life with Amex</a>
+                </div>
+                <div class="column">
+                    <a href="#Membership">Membership Rewards</a>
+                </div>
+            </div>
+            <div class="row">
+                <div class="column">
+                    <!-- there is a register symbol to the right of Express -->
+                    <h2>Get More From American Express</h2>
+                </div>
+                <div class="column">
+                    <a href="url">
+                        <img src="girl-laptop-phone.gif" alt="woman on a laptop and phone resting on ear" style="height:162px;">                
+                    </a>   
+                    <h3>Find Places to Use Your Card</h3>
+                    <!--looks like a button -->
+                    <a href="#Map">View Amex Maps</a>
+                </div>
+                <div class="column">
+                    <a href="url">
+                        <img src="guy-laptop.gif" alt="man on a laptop" style="height:162px;">                
+                    </a>   
+                    <h3>Are you a business Owner?</h3>
+                    <!--looks like a button -->
+                    <a href="#Business">Learn More</a>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-footer">
```

Comment:
col-4 does not make sense here.
It makes sense within the next row (which actually has 4 columns).
But this row (on line 172), has only one column, so its width would not need to be 25% as col-footer is given in your css